Fish-Registration/Views/Shared/_Layout.cshtml

```diff
@@ -73,6 +80,20 @@
         <script src="~/js/site.min.js" asp-append-version="true"></script>
     </environment>
 
+    <!-- Start review stars -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-bar-rating/1.2.1/jquery.barrating.min.js"></script>
```

Comment:
This is fine for now but just as an FYI, this project has `bower`. Although bower is [no longer supported ](https://bower.io/blog/2017/how-to-migrate-away-from-bower/) and is not the preferred way to package front-end code, it is for the purpose of installing front-end packages. So if you have bower-cli installed, you could do something like `bower install jquery-bar-rating` just like you would with `npm`. Then, instead of `package.json` file, it would be added to the `bower.json` file. I believe packages added through bower are added to the HTML file through that `@RenderSection("Scripts", required: false)` statement at the bottom of this file. 

To be clear, no need to change anything here. Just thought I'd let you know in case you didn't notice `bower`.